src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -2130,6 +2132,32 @@ private function addHttpClientSection(ArrayNodeDefinition $rootNode, callable $e
         ;
     }
 
+    private function createHttpClientCachingSection(): ArrayNodeDefinition
+    {
+        $root = new NodeBuilder();
+
+        return $root
+            ->arrayNode('caching')
+                ->info('Caching configuration.')
+                ->canBeEnabled()
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->stringNode('cache')
+                        ->info("The taggable cache's service ID.")
+                        ->defaultValue('cache.app.taggable')
```

Comment:
the default should probably be a dedicated cache pool instead of using `cache.app` directly, to avoid key conflicts (and when configuring scoped clients, they should probably use their own cache pool to avoid sharing cache keys between different configured clients)

Reply:
Adressed in https://github.com/symfony/symfony/pull/59576/commits/ec272b9975e8276382ad997e5ba22830ec359773